Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicRequestHandler.cpp

```diff
@@ -75,5 +76,21 @@ namespace AzToolsFramework
 
             return instantiatePrefabOutcome.GetValue();
         }
+
+        bool PrefabPublicRequestHandler::DeleteEntitiesAndAllDescendantsInInstance(const EntityIdList& entityIds)
+        {
+            auto deleteOutcome = m_prefabPublicInterface->DeleteEntitiesAndAllDescendantsInInstance(entityIds);
+            if (!deleteOutcome.IsSuccess())
+            {
+                AZ_Error("DeleteEntitiesAndAllDescendantsInInstance", false,
+                    "Failed to delete entities and all their descendants in prefab instance. Error message: %s.",
+                    deleteOutcome.GetError().c_str());
+
+                return false;
+            }
+
+            return true;
+        }
```

Comment:
Python can use Outcome objects just fine, so there's no need to wrap these functions in C++.

Reply:
Fixed.